### DISCUSSION TITLE
An error occurred during software installation

### DISCUSSION THREAD

**Discussion by zhangyu3230** [idaholab/moose#21997] [Q&A Getting Started] [answered]
(moose) zhangyu@AZY:/mnt/c/For_MOOSE/projects/moose/test$ make -j 4.3
Command 'make' not found, but can be installed with:
sudo apt install make        # version 4.3-4.1build1, or
sudo apt install make-guile  # version 4.3-4.1build1
(moose) zhangyu@AZY:/mnt/c/For_MOOSE/projects/moose/test$

![Snipaste_2022-09-02_11-44-52](https://user-images.githubusercontent.com/53269552/188055230-7b2fea7c-cfa1-43ba-9683-8e3e463a755f.png)

**Comment by loganharbour** [comment, marked as answer]
What's the result of running `conda list`? Paste text only, please. Do not include screenshots.
